designs/teagan/src/back.mjs

```diff
@@ -36,6 +47,7 @@ function teaganBack({
   paths.seam = new Path()
     .move(points.cfHem)
     .join(paths.hemBase)
+    .join(paths.sideSeam)
```

Comment:
Seam allowance is not being calculated properly for the new side seam. With Curve To Waist disabled, the seam allowance is too close to the fabric edge, and the seam allowance doesn't change when the option is enabled, so it can be too wide.
![Screenshot 2022-12-22 at 5 58 40 AM](https://user-images.githubusercontent.com/109869956/209150299-84f54b36-6ecc-4b57-817a-363b2a86e9d2.png)
![Screenshot 2022-12-22 at 5 58 59 AM](https://user-images.githubusercontent.com/109869956/209150311-6cb6e6e8-e51f-47bc-aec0-4094f08f7a92.png)

Reply:
Thanks for pointing this out, I've fixed it.